src/modules/esl-animate/core/esl-animate.ts

```diff
@@ -0,0 +1,22 @@
+import {ready} from '../../all';
+import {attr, boolAttr, ESLBaseElement} from '../../esl-base-element/core';
+import {ESLAnimateService} from './esl-animate-service';
+
+
+export class ESLAnimate extends ESLBaseElement {
+  static is = 'esl-animate';
+
+  @attr({}) public group: number | false;
+  @boolAttr({}) public repeat: boolean;
+
```

Comment:
I think we should support dynamic attributes chande and some kind of `reanimate` method in case ve add a web component